manager/integration/tests/backupstore.py

```diff
@@ -64,6 +66,10 @@ def set_random_backupstore(request, client):
     elif request.param == "nfs":
         set_backupstore_nfs(client)
         mount_nfs_backupstore(client)
+    elif request.param == "cifs":
+        set_backupstore_cifs(client)
+    elif request.param == "azblob":
```

Comment:
If I run other backup related test cases that manipulate the backup store, like `test_backup_volume_list`, `test_backup_block_deletion`, 
`test_backup_metadata_deletion`, `test_all_replica_restore_failure`, etc., will they pass? If not, should they be skipped?

Reply:
This is exactly what I'm concerned about: https://github.com/longhorn/longhorn-tests/pull/2155#discussion_r1822072923

Reply:
Updated, change `raise NotImplementedError` to  `pytest.skip("Skip test case because the backup store type is not supported") # NOQA` and the teardown function can work after the skip.